packages/react/src/components/ComboButton/ComboButton.stories.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import { MenuItem, MenuItemDivider } from '../Menu';
+
+import { ComboButton } from './';
+import mdx from './ComboButton.mdx';
+
+export default {
+  title: 'Experimental/unstable__ComboButton',
+  component: ComboButton,
+  subcomponents: {
+    MenuItem,
+    MenuItemDivider,
+  },
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Playground = (args) => {
+  const onClick = action('onClick (MenuItem)');
+
+  return (
+    <ComboButton {...args}>
+      <MenuItem label="Second action" onClick={onClick} />
+      <MenuItem label="Third action" onClick={onClick} />
+      <MenuItem label="Fourth action" onClick={onClick} disabled />
+      <MenuItemDivider />
+      <MenuItem label="Danger action" onClick={onClick} kind="danger" />
+    </ComboButton>
+  );
+};
+
+Playground.argTypes = {
+  label: {
+    defaultValue: 'Primary action',
+  },
+};
+
+Playground.args = {
+  onClick: action('onClick'),
+};
```

Comment:
I think we should have a `Default` story on top of the `Playground` and for the Playground we should disable: children, translateWithId, className

Reply:
Do we need to explicitly disable `translateWithId`? I don't see any control for it in the storybook by default.

Reply:
I think we don't want it to show up at all in the playground. See https://deploy-preview-13224--carbon-components-react.netlify.app/?path=/story/components-combobox--playground